src/Uno.Extensions.Reactive/Core/Internal/MessageManager.UpdateTransaction.cs

```diff
@@ -30,62 +30,39 @@ private static class State
 
 		public Message<TResult> Local => _owner.Current;
 
-		public UpdateTransaction(MessageManager<TParent, TResult> owner, CancellationToken ct)
+		internal UpdateTransaction(MessageManager<TParent, TResult> owner, CancellationToken ct)
```

Comment:
Breaking change ?

Reply:
No, it's on an internal class, just making the ctor also internal to make sure that if we make this class public, the ctor should remain internal